tests/mssql-tests.py

```diff
@@ -56,9 +56,10 @@ def main():
         print_usage_error(path.basename(__file__), e)
         exit(2)
     try:
+        # For our tests, tds_version 7.1 is enough
         conn = connect(server=args.server, user=args.username,
                        password=args.password, database=args.database,
-                       port=args.port)
+                       port=args.port, tds_version='7.1')
```

Comment:
This was the problem.

For some reason, at Ubuntu 14.04 and CentOS6, FreeTDS uses at least 7.1 by default, without any extra configuration.

At Ubuntu 16.04 (and I guess 18.04), we need to specify it.

Anyway it doesn't hurt as this is supported since pymssql 2.0.0, and we use at least 2.1.3 at all CI containers.